index.js

```diff
@@ -11,17 +11,23 @@ const processFn = (fn, opts) => function () {
 
 	return new P((resolve, reject) => {
 		args.push(function (err, result) {
-			if (err) {
-				reject(err);
```

Comment:
This means that when there is an error, `multiArgs` is not respected, and the rejection `reason` is the error, and the rest of the callback arguments are lost.